Bottom: 6bfe4c27dfc8facb28d6b0293bdcff126eabb8db
Top:    80910c6c8015d3083457aa03d0e5e58677537a87
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 15:19:24 +0000

lesstif: Use Coord type for line-width, not int


---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 795b9fe..c4b314e 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -52,7 +52,7 @@ typedef struct lesstif_gc_struct
 
   Pixel color;
   const char *colorname;
-  int width;
+  Coord width;
   EndCapStyle cap;
   char xor_set;
   char erase;
